dev-packages/cli/src/commands/check-header.ts

```diff
@@ -294,6 +303,29 @@ function toPrintMessage(result: ValidationResult): string {
     return `${colors[result.severity]} OK`;
 }
 
+function fixViolations(rootDir: string, violations: DateValidationResult[], options: HeaderCheckOptions): void {
+    violations.forEach((violation, i) => {
+        printFileProgress(i + 1, violations.length, `Fix ${violation.file}`);
```

Comment:
I wouldn't use `printFileProgress` here as it always clears the last line so that only the last file will be listed here, e.g., `[3 of 3] Fix /home/martin/Git/OpenSource/eclipse-glsp/glsp/dev-packages/cli/src/util/validation-util.ts` (but the others are not shown). Also we need a line break at the end of it as otherwise the `Do you want to create a commit for the fixed files?` will be shown immediately afterwrads.